core/trino-main/src/main/java/io/trino/exchange/ExchangeManagerRegistry.java

```diff
@@ -104,7 +104,7 @@ public void shutdown()
             }
         }
         catch (Throwable t) {
-            log.error(t, "Error shutting down exchange manager: %s", exchangeManager);
+            log.debug(t, "Error shutting down exchange manager: %s", exchangeManager);
```

Comment:
please revert logging for error situations to `error`/`warn`.
This does not reduce log verbosity in case when cluster is functioning as expected but reduces observability/debuggability if something misbehave

Reply:
See above: https://github.com/trinodb/trino/pull/19554#issuecomment-1790828066